packages/wallets/src/components/WalletsAddMoreCardBanner/WalletsAddMoreCardBanner.tsx

```diff
@@ -81,18 +80,16 @@ const WalletsAddMoreCardBanner: React.FC<TWalletCarouselItem> = ({
     return (
         <div className='wallets-add-more__banner'>
             <div className='wallets-add-more__banner-header'>
-                <span className='wallets-add-more__banner-logo'>
-                    <WalletAddMoreCurrencyIcon currency={currency ? currency.toLowerCase() : ''} />
-                </span>
+                <WalletAddMoreCurrencyIcon currency={currency ? currency.toLowerCase() : ''} />
             </div>
             <WalletButton
                 color='white'
                 disabled={isAdded}
                 icon={
                     isAdded ? (
-                        <CheckIcon className='wallets-add-more__banner-button-icon' width={16} />
+                        <LabelPairedCheckMdFillIcon fill='#333333' />
                     ) : (
-                        <PlusIcon className='wallets-add-more__banner-button-icon' width={16} />
```

Comment:
Shouldn't these unused classes also be cleaned up from the scss file

Reply:
Will make the ammendments 👍🏼